docs/Projects/TAD/tools_comparison.md

```diff
@@ -0,0 +1,228 @@
+# Comparison of tools for transparency of algorithmic decision making
+
+In [tools.md](tools.md) we have researched a few tools which we want to investigate further, this document is the next
+step in that investigation. The checklist we will lay these tools against is written down at [index.md](index.md). The
+Fulfilled column will give a numerical value based on the priority which can then be compared with the other tools per
+section. The scoring is as follows: {M:4, S:3, C:2, W:1}.
```

Comment:
i would make W -1 or -2. The W means it should not be in the product. 
```suggestion
section. The scoring is as follows: {M:4, S:3, C:2, W:-1}.
```

Reply:
Currently we wouldn't ;) have any would not haves in our checklist but I agree with you here about the scoring and will incorporate it